.github/workflows/auto_cherry_pick.yml

```diff
@@ -35,6 +35,39 @@ jobs:
       - name: Cherry pick into ${{ matrix.to_branch }}
         uses: jyejare/github-cherry-pick-action@main
         with:
+          token: ${{ secrets.CHERRYPICK_PAT }}
           branch: ${{ matrix.to_branch }}
           labels: |
             Auto_Cherry_Picked
+            ${{ matrix.to_branch }}
+          assignees: |
+            ${{ github.event.pull_request.user.login }}
+
+  create-issue:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') }}
+    needs: auto-cherry-pick
+    steps:
+      - name: Create Issue on Failed Auto Cherrypick
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ secrets.CHERRYPICK_PAT }}
+          title: "[Failed-AutoCherryPick] - ${{ github.event.pull_request.title }}"
+          body: |
+            #### Auto-Cherry-Pick WorkFlow Failure:
+            - [Failed Cherrypick Action](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+            - [Parent Pull Request](https://github.com/${{ github.repository }}/pull/${{ github.event.number }})
```

Comment:
Adding `${{ matrix.to_branch }}` helps in the issue to know which cherry-pick was failed

Reply:
Good call !

Reply:
Added the to-branch in Description, title and labels.